src/features/upload-documents.feature

```diff
@@ -109,8 +109,11 @@ Feature: upload documents
             # | metric   |
             | land-ownership  |
 
-    Scenario Outline: I can upload all allowed filetypes
-        Then I should be able to upload all allowed filetypes
+    @new
+    Scenario Outline: I should be able to upload a .<filetype> filetype for <document> files
```

Comment:
`I should be able to upload a .` should the `.` be there?

Reply:
looks better when it spits out the test run in the report